payments-core/src/main/java/com/stripe/android/StripeKtx.kt

```diff
@@ -176,15 +177,8 @@ suspend fun Stripe.createAccountToken(
     accountParams: AccountParams,
     idempotencyKey: String? = null,
     stripeAccountId: String? = this.stripeAccountId
-): Token = runApiRequest {
-    stripeRepository.createToken(
-        accountParams,
-        ApiRequest.Options(
-            apiKey = publishableKey,
-            stripeAccount = stripeAccountId,
-            idempotencyKey = idempotencyKey
-        )
-    )
+): Token {
+    return createToken(accountParams, idempotencyKey, stripeAccountId)
```

Comment:
All token methods call the same endpoint with `T : TokenParams`, so unifying this here.